banjo/conf/schema.xml

```diff
@@ -216,6 +216,7 @@
    <!-- link copy to work -->
    <field name="work_id" type="long" indexed="true" stored="true"/>
    <field name="work_pi" type="string" indexed="true" stored="true"/>
+   <field name="work_alias_pi" type="text_general" indexed="true" stored="true" multiValued="true"/>

```

Comment:
@bsingh-nla "alias_pi" is being used as a copy field. "work_alias_pi" is the copy's work's alias pi.